### PR TITLE
feat: add Jaccard similarity evaluator

### PR DESCRIPTION
**Pull request by smy0428**
add Jaccard similarity evaluator for text

### CODE REVIEW

File: evaluators/text/JaccardSimilarityEvaluator/tests/test_jaccardsimilarityevaluator.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+@pytest.mark.parametrize('actual, desired, distance',
+    [
+        ('hello world', 'hello world', 1.0),
+        ('HELLO WORLD', 'hello world', 1.0),
+        ('hello world', 'hello      ', 0.5),
+        ('hello      ', 'hello world', 0.5),
+        ('gdkkm      ', 'hello      ', 0.0),
+        ('hey yup man', 'whats up man', 0.2),
+        ('', '', 0.0),
```
Comment (JoanFM) [inline evaluators/text/JaccardSimilarityEvaluator/tests/test_jaccardsimilarityevaluator.py:14]: what happens if one is empty and not the other?

**Comment by JoanFM** [comment]
Thanks for your contribution. Unfortunately, as you didn't set up your local git config correctly, the author of your commit does not match with your Github account. This poses problems:
Your contribution can not be credited, and will not show up in the list of contributors (as bad git config = anonymous)
We can not accept CLA for anonymous contribution.
Please note that this does not only affect your contribution to Jina, but also affects all your contribution to anything on Github. So better fix it, once for all.
Here is what you can do:
```
git config user.name "Correct new name"
git config user.email "correct@new.email" -> match with your Github registered email address
```
Then fix the commit message by
```
git commit --amend --author="Author Name <email@address.com>" --no-edit
git push --force
```

**Comment by JoanFM** [comment]
recheckcla

**Comment by smy0428** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by JoanFM — APPROVED** [review]

**Pull request merged**